src/application/collections/windows.yaml

```diff
@@ -580,8 +580,8 @@ actions:
                         name: Clear Windows temp files
                         recommend: standard
                         code: |-
-                            del /f /q %localappdata%\Temp\*
-                            del /s /q "%WINDIR%\Temp\*"
+                            del /s /f /q %TEMP%*
```

Comment:
Add missing backslash to path. Should be `%TEMP%\*`

Reply:
Thanks, fixed